CAIPs/caip-294.md

```diff
@@ -138,7 +138,7 @@ The `walletData` object MUST include the following properties:
 
 Additionally, the `walletData` object MAY include the following optional properties:
 
-- `extensionId`: The canonical extension ID of the wallet provider for the active browser.
+- `target`: An object containing `type: caip341` and `value: <extensionId>` used to connect to wallets using `externally_connectable` . For reference on this specification, see [CAIP-341](https://github.com/ChainAgnostic/CAIPs/blob/656551f800843b92243fb08ca6c24e805ad149a3/CAIPs/caip-341.md).
```

Comment:
i think we want to make it a bit more clear that other CAIPs can extend this and that CAIP-341 is an example of a valid target type

Reply:
I believe this point is now made across in the new revision [9623140](https://github.com/pedrouid/CAIPs/pull/9/commits/9623140975cb97d9b80b074da7cad4b1e2eb9b71)